librz/core/cmd/cmd_print.c

```diff
@@ -6483,3 +6252,241 @@ RZ_IPI RzCmdStatus rz_print_instr_until_handler(RzCore *core, int argc, const ch
 	disasm_until_ret(core, core->offset, limit, state->mode);
 	return RZ_CMD_STATUS_OK;
 }
+
+typedef struct rz_core_analysis_stats_range_t {
+	RzCoreAnalysisStats *as;
+	ut64 from;
+	ut64 to;
+	ut64 piece;
+} RzCoreAnalysisStatsRange;
+
+static RzCoreAnalysisStatsRange *analysis_stats_range(RzCore *core, int width) {
+	int cols = rz_config_get_i(core->config, "hex.cols");
+	int w = RZ_MAX(cols, width);
+
+	ut64 from = UT64_MAX;
+	ut64 to = 0;
+
+	RzList *list = rz_core_get_boundaries_prot(core, -1, NULL, "search");
+	if (!list || rz_list_empty(list)) {
```

Comment:
rz_list_empty calls rz_list_length which allows nullable. so you can remove the null check